### PR TITLE
RELATED: RAIL-2820 Add zooming capabilities to DashboardView 

### PR DESCRIPTION
**Pull request by no23reason**
* The icon was moved to sdk-ui-ext
* The positioning of the button was overridden in DashboardView
  to accommodate for slightly different buttons available
* The HighCharts renderer was slightly cleaned up

---

Supported PR commands:

| Command         | Description            |
| --------------- | ---------------------- |
| `ok to test`    | Re-run standard checks |
| `extended test` | BackstopJS tests       |

---

# PR Checklist

-   [ ] commit messages adhere to the [commit message guidelines](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#what-should-the-commits-look-like)
-   [ ] review was done by a Code owner [if necessary](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-tell-if-my-pull-request-needs-approval-by-a-code-owner) (if you think it is not necessary, explain the reasoning in the description or in a comment)
-   [x] `check` passes
-   [x] `check-extended` passes
-   [ ] `rush change` [was run if applicable](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-describe-my-changes-for-the-changelog)

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/1929/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/3557/ : SUCCESS in 14m 08s

**Comment by no23reason** [comment]
extended test

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/1930/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-backstop-tests/566/ : SUCCESS in 30m 23s

**Comment by no23reason** [comment]
extended test sonar

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/1939/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-backstop-tests/569/ : SUCCESS in 17m 41s

File: libs/sdk-ui-charts/src/highcharts/adapter/HighChartsRenderer.tsx [hunk 1]
```diff
@@ -109,6 +104,8 @@ export class HighChartsRenderer extends React.PureComponent<
         this.realignPieOrDonutChart();
     }
 
+    private throttledOnWindowResize = throttle(this.onWindowResize.bind(this), 100);
```
Comment (kandl) [inline libs/sdk-ui-charts/src/highcharts/adapter/HighChartsRenderer.tsx:107]: maybe we should change onWindowResize to arrow function as well? :)

**Review by kandl — APPROVED** [review]
LGTM

File: libs/sdk-ui-charts/src/highcharts/adapter/HighChartsRenderer.tsx (hunk 1 above)
Comment (no23reason) [reply libs/sdk-ui-charts/src/highcharts/adapter/HighChartsRenderer.tsx]: good catch! fixed

**Review by kandl — APPROVED** [review]
LGTM

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-fossa-scanning-zuul-el7/622/ : SUCCESS in 31s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/883/ : SUCCESS in 17s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/3566/ : SUCCESS in 14m 14s

**Pull request merged**